README.rst

```diff
@@ -1,84 +1,123 @@
+
+Run the following installation script in your command line:
 
 .. code::
 
     pip install signalfx_lambda
 
-Configuring the ingest endpoint
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, this function wrapper will send to the ``us0`` realm. If you are
-not in this realm you will need to set the ``SIGNALFX_INGEST_ENDPOINT`` environment
-variable to the correct realm ingest endpoint (``https://ingest.{REALM}.signalfx.com``).
-To determine what realm you are in, check your profile page in the SignalFx
-web application (click the avatar in the upper right and click My Profile).
+Step 2: Locate and set the ingest endpoint
+-------------------------------------------
+By default, this function wrapper will send data to the us0 realm. As a result, if you are not in us0 realm and you want to use the ingest endpoint directly, then you must explicitly set your realm. To set your realm, use a subdomain, such as ingest.us1.signalfx.com or ingest.eu0.signalfx.com.
 
+To locate your realm:
 
-Environment Variables
-~~~~~~~~~~~~~~~~~~~~~
+1. Open SignalFx and in the top, right corner, click your profile icon.
+2. Click **My Profile**.
+3. Next to **Organizations**, review the listed realm.
 
-**Note: the environment variables ``SIGNALFX_INGEST_ENDPOINT`` and ``SIGNALFX_AUTH_TOKEN`` are being deprecated and will not be supported in future releases.**
 
-.. code:: bash
+Step 3: Set environment variables
+----------------------------------
 
-    SIGNALFX_ACCESS_TOKEN=access token
-
-    # endpoint for both metrics and tracer. Overridden by SIGNALFX_METRICS_URL
-    # and SIGNALFX_TRACING_URL if set
-    SIGNALFX_ENDPOINT_URL=http://<my_gateway>:8080
+1. Set SIGNALFX_ACCESS_TOKEN with your correct access token. Review the following example. 
 
-    # optional metrics and tracing configuration
-
-    SIGNALFX_METRICS_URL=ingest endpoint [ default: https://pops.signalfx.com ]
-    SIGNALFX_SEND_TIMEOUT=timeout in seconds for sending datapoint [ default: 0.3 ]
-
-    SIGNALFX_TRACING_URL=tracing endpoint [ default: https://ingest.signalfx.com/v1/trace ]
+.. code:: bash
 
-``SIGNALFX_ENDPOINT_URL`` can be used to configure a common endpoint for metrics and
-traces, as is the case when forwarding with the Smart Gateway. The path ``/v1/trace``
-will automatically be added to the endpoint for traces.
+    SIGNALFX_ACCESS_TOKEN=access token
 
-If either ``SIGNALFX_TRACING_URL`` or ``SIGNALFX_METRICS_URL`` are set, they will take
-precendence over ``SIGNALFX_ENDPOINT_URL`` for their respective components.
 
-For example, if only ``SIGNALFX_ENDPOINT_URL`` is set:
+2. If you use POPS, Smart Gateway, or want to ingest directly from a realm other than us0, then you must set at least one endpoint variable. (For environment variables, SignalFx defaults to the us0 realm. As a result, if you are not in the us0 realm, you may need to set your environment variables.) You can update one of the variables below. Review the following examples.  
 
```

Comment:
Hi George, sorry, I thought I wrote it somewhere;) I would put lines 52-74 back here, just after line 97. The rest should stay as it is.